src/LykkePublicAPI/Models/Trades/TradesResponseModel.cs

```diff
@@ -25,7 +24,7 @@ public static TradesResponseModel ToResponseModel(this Trade trade)
             {
                 Id = trade.Id,
                 AssetPairId = trade.AssetPairId,
-                DateTime = trade.DateTime,
+                DateTime = trade.DateTime.ToString("yyyy-MM-ddTHH:mm:ss.fffZ"),
```

Comment:
Have tried to use IsoDateTimeConverter and its DateTimeFormat propery? If it works, this converter could be set for all API endpoints at once.